Program.cs

```diff
@@ -67,30 +65,24 @@
                 });
 });
 
-builder.Services.AddAuthentication("token")
-                .AddJwtBearer("token", options =>
-                {
-                    //options.Authority = "https://dsf-idsrv-dev.dmrid.gov.cy"; //Urls.CYLogin;
-                    options.Authority = builder.Configuration["IdentityServer:Authority"];
-                    options.TokenValidationParameters.ValidateAudience = false;
+builder.Services
+    .AddAuthentication("token")          
+    .AddJwtBearer("token", options =>
+    {
+        // base-address of Identity Server
+        options.Authority = builder.Configuration["IdentityServer:Authority"];
+        options.TokenValidationParameters.ValidateAudience = false;
 
-                    options.TokenValidationParameters.ValidTypes = new[] { "at+jwt", "JWT" }; // currently CYLogin token type is JWT                    
+        options.TokenValidationParameters.ValidTypes = new[] { "at+jwt", "JWT" }; // currently CYLogin token type is JWT                    
 
-                    if (Boolean.Parse(builder.Configuration["Proxy:ProxyEnabled"]))
-                    {
-                        options.BackchannelHttpHandler = new HttpClientHandler
-                        {
-                            Proxy = new WebProxy(builder.Configuration["Proxy:ProxyAddress"])
-                        };
-                    }
-                });
-
-// require the scope dsf.submission in the access token
-builder.Services.AddAuthorization(options =>
-{
-    options.AddPolicy("submission_access", policy =>
```

Comment:
This might be useful if we need to Authorize specific endpoints with a Policy
For example: 
[Authorize(Policy = "submission_access")]
...